packages/libs/types/src/PactCommand.ts

```diff
@@ -141,28 +141,35 @@ export type NetworkId = string | undefined;
 /**
  * @alpha
  */
-export type ChainId =
-  | ''
```

Comment:
@alber70g do you think we should still accept an empty string as a value for ChainId?